src/applications/yellow-ribbon/components/SearchResult/index.jsx

```diff
@@ -130,20 +130,14 @@ export const SearchResult = ({ school }) => (
             <br />
             (per student, per year)
           </p>
-          <p className="vads-u-margin--0">
-            <span className="sr-only">
-              Maximum Yellow Ribbon funding amount (per student, per year)
-            </span>
-            {deriveMaxAmountLabel(school)}
-          </p>
+          <p className="vads-u-margin--0">{deriveMaxAmountLabel(school)}</p>
```

Comment:
Is the `p` still necessary?

Reply:
It is I believe, yupp! Otherwise there would be no wrapping node for the text and no node for screen readers to identify